nixpkgs/nixpkgs.bzl

```diff
@@ -65,12 +71,12 @@ def _nixpkgs_package_impl(ctx):
     path = ["-I", "nixpkgs={0}".format(ctx.path(ctx.attr.repository).dirname)]
   elif ctx.attr.path:
     path = ["-I", "nixpkgs={0}".format(ctx.attr.path)]
+  elif not ctx.attr.nix_file and not ctx.attr.nix_file_content:
```

Comment:
Use De Morgan's law for a more readable guard.

Reply:
I disagree about readability, but I'm sure that's a matter of personal taste, I'll follow yours for that one.